### PR TITLE
Changed return type from parse_json_response

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Review by rnetser — APPROVED** [review]

File: rosa/cli.py [hunk 1]
```diff
@@ -131,8 +131,9 @@ def execute(command, allowed_commands=None):
                     }}
 
     Returns:
-        list or json: json if json.loads(res.stdout) not fail,
-        else list of output line by line.
+        json or dict: json if json.loads(res.stdout) not fail,
```
Comment (myakove) [inline rosa/cli.py:134]: pleas rephrased, you always return dict
I think we always need to return the same structure in the dict
{"stdout": ......, "stderr": .....}

Comment (myakove) [reply rosa/cli.py:134]: {"out": ......, "err": .....}

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — CHANGES_REQUESTED** [review]

File: rosa/cli.py [hunk 2]
```diff
@@ -108,10 +108,16 @@ def parse_help(rosa_cmd="rosa"):
...
-    except json.decoder.JSONDecodeError:
-        return response.splitlines()
+    def _try_json_load(arg):
+        try:
+            return json.loads(arg)
+        except json.decoder.JSONDecodeError:
+            return arg
+
+    return {
+        "out": _try_json_load(response.stdout),
```
Comment (oharan2) [inline rosa/cli.py:118]: Check separately if any one of stdout/stderr could pass json.loads

**Review by myakove — APPROVED** [review]

**Pull request merged**